api/types/accesslist/convert/v1/review.go

```diff
@@ -76,9 +76,7 @@ func FromReviewProto(msg *accesslistv1.Review) (*accesslist.Review, error) {
 func ToReviewProto(review *accesslist.Review) *accesslistv1.Review {
 	var reviewChanges *accesslistv1.ReviewChanges
 	if review.Spec.Changes.MembershipRequirementsChanged != nil {
-		if reviewChanges == nil {
-			reviewChanges = &accesslistv1.ReviewChanges{}
-		}
+		reviewChanges = &accesslistv1.ReviewChanges{}
```

Comment:
Quite honestly, I'd suppress the linter here instead. This change increases a risk of copy&paste errors for little value.

Reply:
Not sure what you mean? The linter flagged that the if statement was unnecessary because the condition is guaranteed to always be true. 

I don't see the connection between an unnecessary if and the potential for copy paste errors - can you elaborate?

Reply:
- the check for whether reviewChanges is nil is essential to not overwriting an existing value
- the code is a repetitive sequence of similarly-constructed statements
- the potential author who adds a next statement to the mix of existing ones may be carelessly repeating the first one, that doesn’t perform the check
- the default GitHub code review context window may prevent the reviewer from noticing it unless they expand more code around the change.

That said, it’s up to you, I don’t feel strongly about this one.